samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -245,10 +247,11 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                   label={'Passcode'}
                   placeholder={'Enter a meeting passcode'}
                   onChange={(_, newValue) => {
-                    const meetingId = callLocator && 'meetingId' in callLocator ? callLocator.meetingId : '';
+                    const localMeetingId = meetingId ? meetingId : '';
```

Comment:
Is this line needed?

Reply:
this is just undefined check

Reply:
Right - but why have it? You have the check again below when `localMeetingId ?` is called